src/control/server/config.go

```diff
@@ -53,6 +53,13 @@ const (
 type networkProviderValidation func(string, string) error
 type networkNUMAValidation func(string, uint) error
 
+// Configuration information shared with the Agent
```

Comment:
In Go, public variables and functions should have a comment of the form `// Foo description`, where "Foo" matches the public name of the thing.

This is more than just a convention; it's used for automatic generation of package documentation, e.g.: https://godoc.org/github.com/daos-stack/daos/src/control/server#Configuration